### ISSUE TITLE
Stuck migration transaction!

### ISSUE THREAD

**Issue by jmansfie** [Polkadex-Substrate/Ido-Thea-beta-testers-feedback#6]
This has been going for 18 hours now

![image](https://user-images.githubusercontent.com/6218678/162452867-920fb6c8-1feb-4736-b7ac-6c99da0d07bc.png)

Was a migration from 0x539eC04D0DaD19c20932b17Fe8DCCcd5D2d6E16b to esnbKSRnU1WKNgJewte4asokmCWwiKBL2HeVWS3NSYEiZfoYc

**Comment by jmansfie** [comment]
I  have 3 token migrations stuck now. I think it happens when you do multiple one after another without waiting for the previous to finish and they all try to finish in the same block

**Comment by Pastashooter105** [comment]
Tried to bridge 50 FAU. Thea bridge shows random block number and no success, while Etherscan shows 56+ confirmations. 
![screen1](https://user-images.githubusercontent.com/103290654/162542683-92086489-15b7-46b4-bdde-f0f702a9f73f.jpg)
![screen1](https://user-images.githubusercontent.com/103290654/162542683-92086489-15b7-46b4-bdde-f0f702a9f73f.jpg)
![screen2](https://user-images.githubusercontent.com/103290654/162542694-4aefadd5-e867-4aff-b2e0-4e300b12efb3.jpg)
![screen2](https://user-images.githubusercontent.com/103290654/162542694-4aefadd5-e867-4aff-b2e0-4e300b12efb3.jpg)

**Comment by Pastashooter105** [comment]
Tried to bridge 50 FAU. Thea bridge shows random block number and no success, while Etherscan shows 56+ confirmations. 
![screen1](https://user-images.githubusercontent.com/103290654/162542683-92086489-15b7-46b4-bdde-f0f702a9f73f.jpg)
![screen1](https://user-images.githubusercontent.com/103290654/162542683-92086489-15b7-46b4-bdde-f0f702a9f73f.jpg)
![screen2](https://user-images.githubusercontent.com/103290654/162542694-4aefadd5-e867-4aff-b2e0-4e300b12efb3.jpg)
![screen2](https://user-images.githubusercontent.com/103290654/162542694-4aefadd5-e867-4aff-b2e0-4e300b12efb3.jpg)

**Comment by charleslukes** [comment]
> I have 3 token migrations stuck now. I think it happens when you do multiple one after another without waiting for the previous to finish and they all try to finish in the same block
> 
> ![image](https://user-images.githubusercontent.com/6218678/162462277-76997681-bfb8-43e0-993d-fecc548463b7.png)

okay, when you did the multiple transactions did you first confirm and sign each of them before proceeding to the other?

**Comment by Pastashooter105** [comment]
Yes. I always confirmed and signed the bridge transaction and waited till Ethersscan showed at least 20 or >50 confirmations.

---
Markus Batscheider 
Neue Mobilfunknummer ab 01.11.2017:
0173-4032251

**Comment by charleslukes** [comment]
okay thank you for your response, I will try to recreate and fix it.

**Comment by Pastashooter105** [comment]
> okay thank you for your response, I will try to recreate and fix it.
I tested the Token Manager again, thea bridge is working for me now, too

**Comment by Pastashooter105** [comment]
> okay thank you for your response, I will try to recreate and fix it.
I tested the Token Manager again, thea bridge is working for me now, too
